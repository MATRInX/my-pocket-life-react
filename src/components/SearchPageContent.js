import React from 'react';

const SearchPageContent = (props) => (
  <div>
    SearchPageContent content... - query string: {props.match.params.queryString}
  </div>
);

export default SearchPageContent;