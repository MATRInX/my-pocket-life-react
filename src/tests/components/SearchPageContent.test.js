import React from 'react';
import { shallow } from 'enzyme';
import SearchPageContent from '../../components/SearchPageContent';

let wrapper, queryString, match;

beforeEach(() => {
  queryString = 'tekst';
  match = {
    params: {
      queryString
    }
  };
  wrapper = shallow(<SearchPageContent match={match}/>);
})

test('should correctly render SearchPageContent', () => {  
  expect(wrapper).toMatchSnapshot();
});

test('should correctly show typed search word', () => {
  const categoryNode = wrapper.find('p');
  expect(categoryNode.text()).toEqual(queryString);
});