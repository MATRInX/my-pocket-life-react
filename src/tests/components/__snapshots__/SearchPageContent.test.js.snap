// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should correctly render SearchPageContent 1`] = `
<div>
  SearchPageContent content... - query string:
  <p>
    tekst
  </p>
  <Connect(PostList) />
</div>
`;
